TrackletAlgorithm/ProjectionCalculator.h

```diff
@@ -37,43 +39,54 @@ PC::addProj(const TrackletProjection<TProjType> &proj, const BXType bx, Tracklet
   else
     phi >>= 3;
 
+  int ipage = trackletIndex >> 7;
+
+  trackletIndex = trackletIndex&127;
```

Comment:
@aryd And I think you can use `kNBits_MemAddr` to get rid of the magic 127 here:
```suggestion
  trackletIndex = trackletIndex & ((1 << kNBits_MemAddr) - 1);
```

Reply:
Done.